### DISCUSSION TITLE
Yaml code for the mail card

### DISCUSSION THREAD

**Discussion by eigenphase** [moralmunky/Home-Assistant-Mail-And-Packages#685] [Q&A]
Can someone provide me with the YAML code that generates the card in the documentation? https://raw.githubusercontent.com/moralmunky/Home-Assistant-Mail-And-Packages-Custom-Card/master/img/card-image.png

It seems my interactive editor is broken and I need to use Yaml.

**Comment by firstof9** [comment]
That would be @moralmunky's other project over at 
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages-Custom-Card
